### PR TITLE
Support to htmlDecode() w/out html_entity_decode()

### PR DESCRIPTION
**Pull request by odahcam**

### CODE REVIEW

File: src/Stringy.php [hunk 1]
```diff
@@ -477,8 +477,21 @@ public function htmlDecode($flags = ENT_COMPAT)
      * @return Stringy  Object with the resulting $str after being html encoded.
      */
     public function htmlEncode($flags = ENT_COMPAT)
-    {
-        $str = htmlentities($this->str, $flags, $this->encoding);
+    {        
+        if (function_exists('html_entity_decode')) {
```
Comment (danielstjules) [inline src/Stringy.php]: Does this ever evaluate to false? Also, why would `htmlEncode` return the result of `html_entity_decode` ?

**Comment by danielstjules** [comment]
Thanks for the PR! But I think there was some issues applying the patch and decode logic was placed in the htmlEncode method?

File: src/Stringy.php (hunk 1 above)
Comment (odahcam) [inline src/Stringy.php]: Yes! When PHP version is lower than `4.3.0` the `else` block will work as a polyfill.

Comment (odahcam) [inline src/Stringy.php]: Or simply when the function `html_entity_decode` is not supported.

**Comment by odahcam** [comment]
@danielstjules I mistakenly put the code on `Stringy::htmlEncode` method, but I should put this code on `Stringy::htmlDecode` method.

**Pull request closed without merging**